ai_services/anomaly_detection/sensitivity_auto_tuner/standalone_univariate/requirements.txt

```diff
@@ -0,0 +1,4 @@
+pandas
```

Comment:
can you add the specific version to install? such as `pandas ==1.1.0`

Reply:
Resolved